components/grid/grid.css

```diff
@@ -0,0 +1,94 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
+
+body {
+    background-image: none;
+}
+
+.component-container {
+    margin-left: 5rem;
+
+}
+.component-heading{
+    margin-bottom: 1rem;
+}
+.component-description {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    font-size: 1.5rem;
+}
+.grid-container {
+    height: auto;
+    width: 95%;
+    padding: 1rem;
+    margin-bottom: 4rem;
+    margin-top: 2rem;
+    border: 2px solid var(--border-color); 
+
+}
+
+.grid-area1{
+    width: 100%;
+    height: 30vh;
+    padding: 10px;
+    border: 1px solid var(--border-color);
+    display: grid;
+    grid-template-rows: 1fr 1fr ;
+    grid-template-columns: repeat(5, 1fr) ;
+    grid-gap: 1rem;
+    text-align: center;
+}
+.grid-item1{
+    grid-column: 1/3;
+}
+.grid-item3{
+    grid-column: 4/6;
+}
+.grid-item5{
+    grid-column: 2/5;
+}
+
+.grid-area2{
+    width: 100%;
+    height: 40vh;
+    padding: 10px;
+    border: 1px solid var( --border-color);
+    display: grid;
+    grid-template-rows: 1fr 1fr 1fr ;
+    grid-template-columns: repeat(7, 1fr) ;
+    grid-gap: 1rem;
+    text-align: center;
+}
```

Comment:
you can make a common class for grid-area-1 and grid-area-2 and in that use those properties which are common in both of them. Here I can see width, padding, border etc are common.

Reply:
I will refactor  it

Reply:
made the changes